helm/dagger/values.yaml

```diff
@@ -14,12 +15,14 @@ engine:
   #   [log]
   #     format = "json"
   labels: {}
+
   ### Customize Dagger Engine start args
-  # "--oci-max-parallelism num-cpu" is kept as the default behaviour so that we don't surprise users.
-  # You may want to remove it in your deployments. FTR: https://github.com/dagger/dagger/pull/7395
+  #   "--oci-max-parallelism num-cpu" is kept as the default behaviour so that we don't surprise users.
+  #   You may want to remove it in your deployments. FTR: https://github.com/dagger/dagger/pull/7395
   args:
```

Comment:
Now that we are doing a breaking change, should we bite the bullet and remove the default `--oci-max-parallelism` setting we have here? We have been running without it for a while now 

cc @gerhard @jedevc

Reply:
We most definitely should. Would like to keep it as a follow-up PR, so that we don't change too much at once.